src/App.js

```diff
@@ -133,7 +133,7 @@ class App extends React.Component {
       this.state.error_message = this.checkForErrors(this.state.configuration);
     }
 
-    if (window.location.hostname !== 'dashboard.transitmatters.org') {
+    if (window.location.hostname !== 'dashboard-v3.labs.transitmatters.org') {
```

Comment:
May I recommend 
```suggestion
    if (window.location.hostname !== 'dashboard-old.labs.transitmatters.org') {
```
since most people might not realize it was even v3 before